contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,253 @@
+
+- **Improve your product:** If you're talking to your users, and learning about your power users, then you should be well-placed to build awesome new features your existing (and potential) customers will love. I'm told this is "creating value" in the official business lingo.
+
+- **Focus on customer success:** Customer success > outbound sales a lot of the time. [Our customer success team](/handbook/small-teams/customer-success) owns all inbound contact requests, assisting new users and introducing them to the right PostHog people at the right time. If you can't afford a dedicated team, give the job to someone until you do. PostHog CEO, James Hawkins, owned this area in our early days.
+
+- Re-engage your users: Users who like your product can sometimes be distracted by other things. Well-timed re-engagement messages, such as "you left these items in your basket", can be extremely powerful. [Path analysis](/product/user-paths) is a useful tool for understanding where users get distracted.
+
+### The things you need to improve retention
+
+You're gonna need a few things to track, analyze and reduce customer churn, such as:
+
+1. The ability to identify your new/current users (e.g. job role, location etc.)
+
+1. An analytics tool to help you track user churn, and analyze use behavior.
+
+1. Session recording, so you can watch real interactions with your product.
+
+1. A way to test new features and improvements to validate if they work.
+
+1. The ability to safely roll out new features to test groups and specific cohorts
+
+1. Product-minded engineers or product managers who talk to users and ship fast
+
+Some companies deploy a complicated data stack to get all this, but it's not necessary. We built PostHog (yes, this is a teeny bit of sales pitch, now) so you can get all those things in one, tightly integrated tool.
+
+And, while we can't help you with #6, we do have strong opinions on why [product engineers are awesome](/blog/what-is-a-product-engineer) for startups, and [how product engineers and product managers](/blog/product-engineer-vs-product-manager) can work together.
+
+Check out [our product page](/product) if this sounds interesting to you.
+
+/pitch over
+
+## Takeaways
+
+Phew, ok, that was lot. Hopefully you know a little more about how churn and retention rate work, interact, and impact your product now.
+
+To conclude, here are {CHANGE THIS} takeaways you can use to sound smart in meetings, or perhaps share on LinkedIn 😱 with a hashtag – I won't judge, not publicly at least:
+
+1. Early-stage startups shouldn't get too hung up on how high or low churn is, but you should care **which direction** it's headed. The actual rate of churn / retention is more important once you have product-market fit and care about revenue.
```

Comment:
Should they care about this? There is an argument that they shouldn't care which direction churn is moving in. If they are growing fast, and churn is growing slower, that can still be a good situation.

Reply:
I might caveat this slightly, but I think it's a matter of degrees. Very slow increase in churn might be fine, but a faster rate would be bad and wipe out that growth quite quickly. It's more like "keep an eye on it and don't let it go crazy".